canonical/canonical-indico-themes
# Update call for abstracts page

### Done

Update call for abstracts page & form

### QA
To QA you will need to open 'call for abstracts' (CfA). This can be found in the event management section of the running Indico Instance.
Under 'Workflows', select 'Call for Abstracts' and click 'Open'.
Now go back to the page and you will see 'Call for Abstracts' in the side column.
- Check the 'CfA' landing page.
- Try to submit an abstract and test the different functionalities of the form.

### Fixes
https://github.com/canonical/web-squad/issues/5690
